[2024-04-27T00:00:08.622+0000] {processor.py:161} INFO - Started process (PID=38) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:08.626+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:00:08.635+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:00:08.633+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:08.688+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:00:08.678+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:00:08.690+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:08.756+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.152 seconds
[2024-04-27T00:00:39.410+0000] {processor.py:161} INFO - Started process (PID=97) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:39.412+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:00:39.415+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:00:39.414+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:39.439+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:00:39.434+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:00:39.442+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:00:39.464+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.060 seconds
[2024-04-27T00:01:10.053+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:10.054+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:01:10.057+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:01:10.057+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:10.078+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:01:10.073+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:01:10.079+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:10.095+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.048 seconds
[2024-04-27T00:01:40.462+0000] {processor.py:161} INFO - Started process (PID=215) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:40.463+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:01:40.465+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:01:40.465+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:40.486+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:01:40.481+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:01:40.487+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:01:40.504+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.048 seconds
[2024-04-27T00:02:10.645+0000] {processor.py:161} INFO - Started process (PID=274) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:10.646+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:02:10.647+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:02:10.647+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:10.667+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:02:10.662+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:02:10.668+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:10.688+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.049 seconds
[2024-04-27T00:02:41.005+0000] {processor.py:161} INFO - Started process (PID=333) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:41.006+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:02:41.008+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:02:41.008+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:41.023+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:02:41.020+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:02:41.024+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:02:41.045+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.047 seconds
[2024-04-27T00:03:11.356+0000] {processor.py:161} INFO - Started process (PID=392) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:11.358+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:03:11.359+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:03:11.359+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:11.380+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:03:11.376+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:03:11.382+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:11.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.048 seconds
[2024-04-27T00:03:41.776+0000] {processor.py:161} INFO - Started process (PID=452) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:41.777+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:03:41.779+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:03:41.779+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:41.795+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:03:41.792+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:03:41.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:03:41.813+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.044 seconds
[2024-04-27T00:04:12.137+0000] {processor.py:161} INFO - Started process (PID=511) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:12.138+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:04:12.140+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:04:12.139+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:12.158+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:04:12.155+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:04:12.160+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:12.181+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.051 seconds
[2024-04-27T00:04:42.450+0000] {processor.py:161} INFO - Started process (PID=570) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:42.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:04:42.453+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:04:42.452+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:42.472+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:04:42.468+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:04:42.473+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:04:42.495+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.051 seconds
[2024-04-27T00:14:05.815+0000] {processor.py:161} INFO - Started process (PID=38) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:05.816+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:14:05.818+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:14:05.818+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:05.843+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:14:05.838+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:14:05.844+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:05.868+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.059 seconds
[2024-04-27T00:14:36.400+0000] {processor.py:161} INFO - Started process (PID=97) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:36.401+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:14:36.403+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:14:36.403+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:36.422+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:14:36.417+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:14:36.424+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:14:36.447+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.054 seconds
[2024-04-27T00:15:06.716+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:06.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:15:06.725+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:15:06.722+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:06.746+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:15:06.741+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:15:06.749+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:06.792+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.094 seconds
[2024-04-27T00:15:36.938+0000] {processor.py:161} INFO - Started process (PID=215) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:36.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:15:36.941+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:15:36.941+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:36.959+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:15:36.956+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:15:36.961+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:15:36.981+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.049 seconds
[2024-04-27T00:16:07.378+0000] {processor.py:161} INFO - Started process (PID=274) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:07.379+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:16:07.382+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:16:07.381+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:07.400+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:16:07.395+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:16:07.401+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:07.420+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.049 seconds
[2024-04-27T00:16:37.649+0000] {processor.py:161} INFO - Started process (PID=333) to work on /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:37.650+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/mi_dag.py for tasks to queue
[2024-04-27T00:16:37.652+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:16:37.652+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:37.672+0000] {logging_mixin.py:188} INFO - [2024-04-27T00:16:37.668+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/mi_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/mi_dag.py", line 10, in <module>
    from Primer_entregable import cargar_datos_spotify, cargar_datos_lastfm
ModuleNotFoundError: No module named 'Primer_entregable'
[2024-04-27T00:16:37.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/mi_dag.py
[2024-04-27T00:16:37.692+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/mi_dag.py took 0.048 seconds
